specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/sqlServer.json

```diff
@@ -798,6 +798,244 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/encryptionProtector/{encryptionProtectorName}": {
```

Comment:
As per convention, name should be pluralized. I understand you are planning to expose a singleton resource by name 'current'. In future, this may change. Please use name 'encryptionProtectors'

Reply:
This is a partner (SQL) dependant API that we're exposing from Synapse. We're simply proxying the request as-is to the partner. Hence we're unable to change this.